2-mandatory/3-bush-berries.js

```diff
@@ -23,11 +23,19 @@
 
 function isBushSafe(berryArray) {
   //Write your code here
-}
+   if (berryArray.every(v => v == "pink")) {
+    return "Bush is safe to eat from";
+  } else {
+    return "Toxic! Leave bush alone!";
+  } 
```

Comment:
```suggestion
   if (berryArray.every(v => v == "pink")) {
    return "Bush is safe to eat from";
  }
  return "Toxic! Leave bush alone!"; 
```
We can skip the else statement for a cleaner code